.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## PR Title :boom:
+
+Please title this PR with a summary of the change, along with the JIRA card number.
+
+Suggested formats: 
+
+1. Fixes/Refs #RHIROS-XXX - Title
+2. RHIROS-XXX Title 
+
+Feel free to remove this section from PR description once done.
+
+## Why do we need this change? :thought_balloon:
+
+Please include the __context of this change__ here.
+
+## Documentation update? :memo:
+
+- [ ] Yes
+- [ ] No
```

Comment:
@saltgen based on my experience, very rarely we make any docs update, may be we can have a common section instead of this something like `requires updates`?

Reply:
I agree, but I think this section would act like a gentle reminder. I have updated the section to `Documentation requires update?`

Reply:
got it thanks @saltgen 👍🏻